proxy/src/registry.rs

```diff
@@ -369,6 +384,17 @@ impl Registry {
             settlement: 9,
         }
     }
+
+    /// Returns the cost estimate of a transaction which includes the given messages.
+    pub async fn transaction_costs(
+        &self,
+        _messages: Vec<Message>,
+    ) -> Result<TransactionCosts, error::Error> {
+        Ok(TransactionCosts {
+            deposit: 1000,
+            fee: 100,
```

Comment:
Is this function meant to give an estimate of how much (minimally speaking) the user should specify for each of the tx costs? If so, the doc should better reflect that. Otherwise, it creates confusion since the `fee` is something the user defines, not us. We really just define a minimum (global, transaction-agnostic) value to avoid the network from being DDoSed/spammed.

Reply:
Note that we could add a query to the registry client giving us here that minimum fee value.

Reply:
Created radicle-dev/radicle-registry#324 as a follow-up and addressed the doc string.

Reply:
@xla Done: https://github.com/radicle-dev/radicle-registry/pull/326

Will leave the integration up to you since you are still on it.

Reply:
Fixed by https://github.com/radicle-dev/radicle-registry/pull/326. You can use `client::MINIMUM_FEE` to determine whether a given amount is sufficient and/or to just get the minimal allowed.